components/telemetry-operator/controller/logpipeline/controller_test.go

```diff
@@ -305,7 +252,7 @@ var _ = Describe("LogPipeline controller", func() {
 
 			Expect(k8sClient.Delete(ctx, logPipeline)).Should(Succeed())
 
-			// Fluent Bit daemon set should rollout-restarted (generation changes from 1 to 2)
+			// Fluent Bit daemon set should rollout-restarted (generation is 1)
```

Comment:
This comment seem to be wrong now. We validate that a DaemonSet has been created, right?

Reply:
In the code it validating the generation but yeah gen 1 means that it has been created and not updated